### ISSUE TITLE
`no-manually-wrapped-comments`: false positive when a line ends with a pictographic symbol

### ISSUE THREAD

**Issue by sergiocarneiro** [sindresorhus/eslint-plugin-unicorn#3458]
These two comments get flagged as one wrapped sentence:

```js
// ⚠ Warning callout ⚠
// This is some extra information
```

They're really two separate sentences — the first just happens to end with a symbol (⚠) instead of a period.

I'd expect a line ending in a pictographic symbol (⚠, ✅, 🚀) to be treated as terminated, like normal punctuation.
